src/Options.php

```diff
@@ -14,15 +14,13 @@
  */
 final class Options
 {
-    public const DIST_LOCK_FILENAME = 'dist.lock';
-    public const MERGE_PLAN_FILENAME = 'merge_plan.php';
-    public const DEFAULT_CONFIGS_DIRECTORY = 'config/packages';
+    public const MERGE_PLAN_FILENAME = 'MERGE_PLAN.php';
```

Comment:
Isn't it better to use lower case, since linux is strict with the names? `merge_plan.php`

Reply:
It would not be good if no directory is indicated in the composer.json or in the `Config::class` constructor assume config as the main directory, for example if in the constructor set:

```php
$config = new Config(
    new ConfigPaths(diname(__DIR__,2) . '/config/packages',
    $this->environment,
    [
        'params',
        'events',
        'events-web',
        'events-console',
    ],
);
```

Directory its: `/config/config/packages`

Reply:
> Isn't it better to use lower case, since linux is strict with the names? `merge_plan.php`

`MERGE_PLAN.php` made in capital letters due to the fact that it is now located in the root of `config` and it is immediately clear that this is not a custom config. In lowercase, it is easier to delete it by mistake.

I don't see a problem with Linux, since when reading it, it is also requested in uppercase.

Reply:
In linux the convention for system/hidden/etc. files is to start their names within a dot: `.merge_plan.php`. But I don't think that file name case or a dot in it will bring some clearness about the file's mission to users. A corresponding comment inside will do it much better.